config/locales/en.yml

```diff
@@ -1181,6 +1193,8 @@ en:
         new_password: New password
         info_header: Edit name and e-mail
         password_confirmation: *password_confirmation
+
+    shared:
```

Comment:
`member_photo:pic_header` should go under `devise:registrations:edit`
(The organization is about the action, not the partial file name)

Reply:
Done.